### ISSUE TITLE
[readthedocs] Missing Day of month with ordinal

### ISSUE THREAD

**Issue by vcarel** [crsmithdev/arrow#243]
I noticed Arrow support "Day of month with ordinal", just like [Moment.js](http://momentjs.com/docs/#year-month-and-day-tokens)

| Input | Example | Description |
| --- | --- | --- |
| Do | 1st..31st | Day of month with ordinal |

This is not mentioned in the [Tokens] list (http://arrow.readthedocs.org/en/latest/#tokens) on Readthedocs however.

It should be an easy fix :-)

**Comment by andrewelkins** [comment]
Agreed

**Comment by systemcatch** [comment]
It's in the token list now so this can be closed.

**Comment by aceisace** [comment]
@systemcatch 
First of all, thanks for your great project :)
Sorry to add a comment event though this issue is closed, but since it is related to the day of the month ordinal, I thought it would be a better not to open up a new issue for that alone. 

The problem I'm facing is that the 'Do' token doesn't give any correct output. For example, if I try the following:
```
import arrow
now = arrow.now()
now.format('dddd, Do MMMM YYYY')
```
It returns: 
`Monday, 13o May 2019`
_not_ 
`Monday, 13th May 2019`

It seems as though the 'D' is being recognised correctly but the 'o' is treated as a literal 'o', not part of the Token. I'm not sure if it's a bug or if something has changed, so I wanted to ask how the day of the month ordinal can be displayed correctly.

Thanks in advance

**Comment by systemcatch** [comment]
@aceisace Sorry, didn't see your message, I'm glad you find it useful.

What version of arrow are you using? I get the following on 0.14.2

```shell
>>> import arrow
>>> now=arrow.now()
>>> now.format('dddd, Do MMM YYYY')
'Saturday, 6th Jul 2019'
```

**Comment by aceisace** [comment]
@systemcatch 
No problem. 
Here's what I get on arrow when I run `pip3 show arrow`:
```
Name: arrow
Version: 0.4.2
Summary: Better dates and times for Python
Home-page: http://crsmithdev.com/arrow
Author: Chris Smith
Author-email: crsmithdev@gmail.com
License: Apache 2.0
Location: /home/pi/.local/lib/python3.5/site-packages
Requires: python-dateutil
```

But the output remains the same:
```
>>> import arrow
>>> now=arrow.now()
>>> now.format('dddd, Do MMM YYYY')
'Monday, 8o Jul 2019'
```
Basically, the 'th/st/rd' after the day of month is replaced with a literal letter 'o'.

**Comment by jadchaar** [comment]
Hey @aceisace, it seems that you are on version `0.4.2` and not the latest version `0.14.2`. Try upgrading `arrow` to fix this issue by running the following command: `pip3 install -U arrow`.

Let us know if you are still having issues! Thanks!

**Comment by aceisace** [comment]
@jadchaar 
Thanks for your help. I've managed to get it working. It seems that the installation of arrow on a raspberry pi defaults to `0.4.2`. Using the `-U` parameter managed to solve it.

**Comment by jadchaar** [comment]
That is odd. So each time you did `pip install arrow`, it installed `0.4.2`? How long ago did you originally install `arrow`? What is the output of `pip --version`?

Just trying to understand why this is ocurring :).

**Comment by aceisace** [comment]
@Jadchaar
Yes, that is correct. Installing arrow via pip(3) on a Raspberry Pi defaults to 0.4.2. Last time I installed arrow was a few weeks ago, but I‘ve also installed it a few more times over the last few months. The version remained the same.

This is what I get when I run ‘pip3 --version’ :
pip 9.0.1 from /usr/lib/python3/dist-packages (python 3.5)

With pip -- version:
pip 9.0.1 from /usr/lib/python2.7/dist-packages (python 2.7)

**Comment by jadchaar** [comment]
Ok now that is peculiar. I wonder if it may be because of the odd `arrow` versioning scheme (e.g. `pip` thinks the `1` in `0.14.2` is less than the `4` in `0.4.2`).

@systemcatch it may be worth transitioning `arrow` to the [semantic versioning system](https://semver.org/) once we ship the `get` fixes and `dst` additions. Maybe when we drop `py27` support?
